runatlantis.io/blog/2024/integrating-atlantis-with-opentofu.md

```diff
@@ -0,0 +1,181 @@
+```
+**7** - Last but not least, here we specify which Atlantis-side configurations we will have for the repositories.
+```
+# repository config
+repoConfig: |
+  ---
+  repos:
+  - id: /.*/
+    apply_requirements: [approved, mergeable]
+    allow_custom_workflows: true
+    allowed_overrides: [workflow, apply_requirements, delete_source_branch_on_merge]
+```
+**8** - Configure Atlantis webhook ingress, in the example below we are using the AWS ALB.
+```
+# ingress config
+ingress:
+  annotations:
+    alb.ingress.kubernetes.io/backend-protocol: HTTP
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:certificate
+    alb.ingress.kubernetes.io/group.name: external-atlantis
+    alb.ingress.kubernetes.io/healthcheck-path: /healthz
+    alb.ingress.kubernetes.io/healthcheck-port: "80"
+    alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/ssl-redirect: "443"
+    alb.ingress.kubernetes.io/success-codes: "200"
+    alb.ingress.kubernetes.io/target-type: ip
+  apiVersion: networking.k8s.io/v1
+  enabled: true
+  host: atlantis.your.domain
+  ingressClassName: aws-ingress-class-name
+  path: /*
+  pathType: ImplementationSpecific
+```
+Save all changes made to ```values.yaml```
+
+**9** - Using one of the Atlantis options custom workflows, we can create a file ```atlantis.yaml``` in the root folder of your repository, the example below should meet most scenarios, adapt as needed.
+```
+version: 3
+automerge: true
+parallel_plan: true
+parallel_apply: false
+projects:
+- name: terragrunt
+  dir: .
+  workspace: terragrunt
+  delete_source_branch_on_merge: true
+  autoplan:
+    enabled: false
+  apply_requirements: [mergeable, approved]
+  workflow: terragrunt
+workflows:
+  terragrunt:
+    plan:
+      steps:
+      - env:
+          name: TF_IN_AUTOMATION
+          value: 'true'
+      - run: find . -name '.terragrunt-cache' | xargs rm -rf
+      - run: terragrunt init -reconfigure
+      - run:
+          command: terragrunt plan -input=false -out=$PLANFILE
+          output: strip_refreshing
+    apply:  
+      steps:      
+        - run: terragrunt apply $PLANFILE
+```
+**10** - Now let’s go to the installation itself, search for the available versions of Atlantis:
+```
+helm search repo runatlantis
+```
+Replace ```CHART-VERSION``` with the version you want to install and run the command below:
+
+```
+helm upgrade -i atlantis runatlantis/atlantis --version CHART-VERSION -f values.yaml --create-namespace atlantis
+```
+
+Now, see as configure Atlantis [webhook on github](https://www.runatlantis.io/docs/configuring-webhooks.html?source=post_page-----85ca0fbe45e5--------------------------------#github-github-enterprise) repository.
+
+See as Atlantis [work](https://www.runatlantis.io/docs/using-atlantis.html?source=post_page-----85ca0fbe45e5--------------------------------).
```

Comment:
updated to use native link refs via https://github.com/runatlantis/atlantis/commit/f818292ef2144c881776fb760b831cf181e988da